docs/usage/validators.md

```diff
@@ -204,13 +204,13 @@ to set a dynamic default value.
 ```py test="xfail - we need default value validation"
 from datetime import datetime
 
-from pydantic import BaseModel, validator
+from pydantic import BaseModel, field_validator
 
 
 class DemoModel(BaseModel):
     ts: datetime = None
 
-    @validator('ts', pre=True, always=True)
+    @field_validator('ts', pre=True, always=True)
```

Comment:
`pre` and `always` are not valid in `field_validator`

Reply:
Reverted: https://github.com/pydantic/pydantic/pull/5712/commits/0d7cfa6543a9fe3a50a91cfe6f5acca1c95a8176